tests/serial/map-initial-data/zero-data/SerialWrite.xml

```diff
@@ -0,0 +1,37 @@
+<?xml version="1.0" encoding="UTF-8" ?>
+<precice-configuration>
+  <data:scalar name="Data" />
+
+  <mesh name="MeshOne" dimensions="2">
+    <use-data name="Data" />
+  </mesh>
+
+  <mesh name="MeshTwo" dimensions="2">
+    <use-data name="Data" />
+  </mesh>
+
+  <participant name="One">
+    <provide-mesh name="MeshOne" />
+    <receive-mesh name="MeshTwo" from="Two" />
+    <write-data name="Data" mesh="MeshOne" />
+    <mapping:nearest-neighbor
+      direction="write"
+      from="MeshOne"
+      to="MeshTwo"
+      constraint="consistent" />
+  </participant>
+
+  <participant name="Two">
+    <provide-mesh name="MeshTwo" />
+    <read-data name="Data" mesh="MeshTwo" />
+  </participant>
+
+  <m2n:sockets acceptor="One" connector="Two" />
+
+  <coupling-scheme:serial-explicit>
+    <participants first="One" second="Two" />
+    <max-time-windows value="1" />
+    <time-window-size value="1.0" />
+    <exchange data="Data" mesh="MeshTwo" from="One" to="Two" initialize="true" substeps="false" />
```

Comment:
Could it be valuable to configure at least one of the `zero-data` cases using `initialize=false`. Should have the same effect, right?

Reply:
These are covered by the `map-if-necessary` cases, which always use `initialize=false` and never write data, thus producing only 0 data samples.